.github/workflows/publish-ghcr.yaml

```diff
@@ -0,0 +1,37 @@
+name: Docker Image for sbi toolkit
+
+on:
+  push:
+    branches:
+      - main
+      - provide-a-devcontainer-to-be-used-in-codespace
```

Comment:
left over from testing or do we need to keep this branch?

Reply:
fixed in [1d615c8](https://github.com/sbi-dev/sbi/pull/1070/commits/1d615c8111d57359955e0ed949968c2dda3ede8e)